### DISCUSSION TITLE
Howto: Link Embed Images + Text

### DISCUSSION THREAD

**Discussion by ghost** [EddieHubCommunity/support#1780] [Show and tell] [answered]
How to get a personalized embed image and text when someone posts your link?

**Comment by ghost** [comment, marked as answer]
Open Graph: https://ogp.me/
Twitter Cards (Optional, Inherits From Open Graph if not present): https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary

**Reply by kaiwalyakoparkar** [reply]
Thanks for sharing

**Reply by ghost** [reply]
Thanks for asking: 
![image](https://user-images.githubusercontent.com/76587897/120439709-6fa1f200-c3a0-11eb-9882-7f7acdac54de.png) 😮🥳

**Reply by kaiwalyakoparkar** [reply]
Great! I think that comes on your profile right?

**Reply by ghost** [reply]
Yes
